### PR TITLE
Added network proposal client

### PR DESCRIPTION
**Pull request by teclator**
- https://trello.com/c/XDgl2dj7/816-13-casp-new-all-in-one-dialog
- Main PR: https://github.com/yast/yast-installation/pull/487

### CODE REVIEW

File: src/lib/installation/clients/network_proposal.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      {
+        "rich_text_title" => _("Network Configuration"),
+        "menu_title"      => _("Network Configuration"),
+        "id"              => "network"
+      }
+    end
+
+    def make_proposal(_)
+      {
+        "preformated_proposal" => Yast::Lan.Summary("summary"),
```
Comment (mvidner) [inline src/lib/installation/clients/network_proposal.rb]: preformat**t**ed_proposal

Comment (teclator) [inline src/lib/installation/clients/network_proposal.rb]: Fixed.

**Comment by mvidner** [comment]
For reference, this uses the new API from https://github.com/yast/yast-network/pull/489
Thanks! The code looks good, except...

I think we should put network_proposal to yast2-network.rpm where it used to live, consistently with all other foo_proposal clients in their yast2-foo pacakges.

**Comment by teclator** [comment]
It make sense, moving it.

**Comment by teclator** [comment]
Moved to yast2-network repository yast/yast-network#491

**Pull request closed without merging**